skytemple_files/patch/patches.py

```diff
@@ -92,7 +92,7 @@ class PatchType(Enum):
     EXTRACT_MOVE_CODE = ExtractMoveCodePatchHandler
     EXTRACT_ITEM_CODE = ExtractItemCodePatchHandler
     EXTRACT_SP_CODE = ExtractSPCodePatchHandler
-    MOVE_GROWTH = MoveGrowthPatchHandler
+    #MOVE_GROWTH = MoveGrowthPatchHandler
```

Comment:
and this probably too?

Reply:
I removed it from the list because it depends on a patch that won't be merged until 1.4.